### DISCUSSION TITLE
Can be this module be used for a store with about 40000 items

### DISCUSSION THREAD

**Discussion by ufoloko** [HotcakesCommerce/hotcakes-commerce-core#390] [General]
I ask if this module can be used for a store with about 40000 items.

I need to sync those items with a local system, so i need to run the upgrades for the products directly into database or maybe some api.

Please confirm

Thanks

**Comment by WillStrohl** [comment]
There isn't a built-in or known limitation when it comes to the number of any entity, including the number of products.  It really depends on how the store is configured, how DNN is configured, the specific configuration of your catalog, and the resources available to the website.  

When version 3.7+ is released, whatever that limitation is found to be in each respective environment is, that number is expected to get much higher.
